### PR TITLE
dev-java/trident: min java 1.8, swt:4.10

### PR DESCRIPTION
**Pull request by vaukai**
Package-Manager: Portage-3.0.18, Repoman-3.0.2
Signed-off-by: Volkmar W. Pogatzki <gentoo@pogatzki.net>

### CODE REVIEW

File: dev-java/trident/trident-1.3-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RESTRICT="test"
+
+CDEPEND="dev-java/swt:4.10"
+
+DEPEND=">=virtual/jdk-1.8:*
+	app-arch/unzip
```
Comment (fordfrog) [inline dev-java/trident/trident-1.3-r1.ebuild:24]: proly should go to `BDEPEND` instead.

Comment (vaukai) [reply dev-java/trident/trident-1.3-r1.ebuild]: of course, changed

**Comment by fordfrog** [comment]
@vaukai thanks! it did not compile for me though so i had to fix it in f1f7e7e2b8ada6ae59377d55d7416df77c80168f. re-keyword req should be filed because of dropped `x86`. after rekeyworded it can be stabilized if no issues till then.

**Comment by vaukai** [comment]
> @vaukai thanks! it did not compile for me though so i had to fix it in [f1f7e7e](https://github.com/gentoo/gentoo/commit/f1f7e7e2b8ada6ae59377d55d7416df77c80168f). re-keyword req should be filed because of dropped `x86`. after rekeyworded it can be stabilized if no issues till then.

Sorry to hear you had to rework. No idea why it compiled here :-( 
kw request added to https://bugs.gentoo.org/790761

**Pull request closed without merging**